app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -50,6 +51,11 @@ BUILD_PREREQUIRES="
 
 defineDebugInfoPackage wordgrinder$secondaryArchSuffix \
 	$commandBinDir/wordgrinder
+PATCH()
+{
+	sed -i -e "s|/etc/dictionaries-common/words|$dataDir/words|;" \
+		src/lua/addons/spillchocker.lua
```

Comment:
wordgrinder will actually search for a dictionary called "words" here (check the provided screenshots in the PR)?

Reply:
I used the next line, @fbrosson is the sed expert here though ...

> sed -i -e "s|/etc/dictionaries-common/words|$dataDir/words|*.dictionaries;"

Reply:
jikes, my bad:

>> sed -i -e "s|/etc/dictionaries-common/words|$dataDir/words/*.dictionaries|;"

Reply:
I might be wrong (as I did not test anything yet), but I WordGrinder might accept a directory path instead of the full path of a single directory. If this is true, the I think this could work:
```
PATCH()
{
	sed -i -e "s|/etc/dictionaries-common/words|$dataDir/wordgrinder/dictionaries/|;" \
		src/lua/addons/spillchocker.lua
}
```
It will tell WordGrinder to look for one (or all) dictionary in `/system/data/wordgrinder/dictionaries/`. So adding this in INSTALL;
```
	install -d $dataDir/wordgrinder/dictionaries
	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionaries
```
should populate that directory.

It would be nice, however, to split these files into a separate sub-package (created in the same recipe) that we could call wordgrinder_dictionaries or wordgrinder_words (or anything else).

Reply:
@fbrosson my guess here is that wordgrinder searches for the "dictionaries" instead of a "directory" in the sed commit?

also:

> install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary ;)

![wordgrinder3](https://user-images.githubusercontent.com/16057090/34341417-517e2000-e997-11e7-99ea-439cb32ba9df.png)

Reply:
forget the last thing I posted, still had the wordgrinder settings file installed :)

Reply:
OK, I see. So it seems it's only on Windows that it accepts a directory.

Reply:
I'll try it?

Reply:
I'll try it?

Reply:
This should do it @fbrosson @localanu 

  sed -i -e "s|/etc/dictionaries-common/words|$dataDir/wordgrinder/dictionaries/|;" \
  install -t $dataDir/wordgrinder/dictionaries extras/*.dictionary

Reply:
Oops, I had not noticed I had written \*.dictionar**ies** instead of \*.dictionar**y**.
So thanks @Begasus!

Reply:
@fbrosson any example that we could use for a seperate packages for the directories?

Reply:
Got it :+1:

Reply:
file is a good example.
Adding this after REQUIRES and before PATCH:
```
if [ -n "$secondaryArchSuffix" ]; then
SUMMARY_dictionaries="Built-in dictionaries for WordGrinder"
ARCHITECTURES_dictionaries="any"
PROVIDES_dictionaries="
	wordgrinder_dictionaries = $portVersion
	"
REQUIRES_dictionaries="
	wordgrinder == $portVersion base
	"
SUPPLEMENTS_dictionaries="
	wordgrinder
	"
fi
```
And then inserting this in INSTALL:
```
if [ -n "$secondaryArchSuffix" ]; then
	install -d $dataDir/wordgrinder/dictionaries
	install -t $dataDir/wordgrinder/dictionaries extras/*.dictionaries
	packageEntries dictionaries \
		$dataDir
fi
```
shoud make the dictionaries available as a sub-package.
Remark: arch-indepedent packages cannot be built on secondary arch builds.